### PR TITLE
Allow multi-line comments

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: app/javascript/retrospring/features/answerbox/comment/new.ts [hunk 1]
```diff
@@ -16,7 +16,9 @@ export function commentCreateHandler(event: KeyboardEvent): boolean {
   const counter = document.querySelector(`#ab-comment-charcount-${id}`);
   const group = document.querySelector(`[name=ab-comment-new-group][data-a-id="${id}"]`);
 
-  if (event.which === 13) {
+  console.debug("comment create", event.ctrlKey, event.metaKey, event.which);
```
Comment (pixeldesu) [inline app/javascript/retrospring/features/answerbox/comment/new.ts]:
```suggestion
```

**Review by pixeldesu — CHANGES_REQUESTED** [review]

File: app/javascript/retrospring/features/answerbox/comment/new.ts (hunk 1 above)
Comment (raccube) [reply app/javascript/retrospring/features/answerbox/comment/new.ts]: fc90b74

File: app/views/answerbox/_comments.html.haml [hunk 2]
```diff
@@ -25,10 +25,18 @@
...
     name: "ab-comment-new-group",
     data: { a_id: a.id, controller: "character-count", character_count_max_value: 512 }
   }
-    %input.form-control.comment__input{ type: :text, placeholder: t(".placeholder"), name: "ab-comment-new", data: { a_id: a.id, "character-count-target": "input" } }
-    %span.text-muted.form-control-feedback.comment__character-count{ id: "ab-comment-charcount-#{a.id}", data: { "character-count-target": "counter" } } 512
-    %button.btn.btn-primary.d-none{ type: :button, name: "ab-comment-new-submit", data: { a_id: a.id, "character-count-target": "action" } }= t(".action")
+    .form-group.has-feedback.comment__input-group.input-group
+      %textarea.form-control.comment__input{ type: :text, placeholder: t(".placeholder"), name: "ab-comment-new", data: { a_id: a.id, "character-count-target": "input" } }
+    .comment__submit-wrapper
+      %button.btn.btn-primary{
```
Comment (nilsding) [inline app/views/answerbox/_comments.html.haml:35]: there seems to not be an event hooked to this button -- so posting with either ⌃⏎ or ⌘⏎ works but not via the button (which is probably required on phones)

**Review by nilsding — CHANGES_REQUESTED** [review]

File: app/views/answerbox/_comments.html.haml (hunk 2 above)
Comment (raccube) [reply app/views/answerbox/_comments.html.haml:35]: da18674

**Review by nilsding — APPROVED** [review]

**Review by pixeldesu — APPROVED** [review]

**Pull request merged**